src/diff/props.js

```diff
@@ -73,7 +73,7 @@ function setProperty(dom, name, value, oldValue, isSvg) {
 	else if (name[0]==='o' && name[1]==='n') {
 		let useCapture = name !== (name=name.replace(/Capture$/, ''));
 		let nameLower = name.toLowerCase();
-		name = (nameLower in dom ? nameLower : name).substring(2);
+		name = (nameLower in window ? nameLower : name).substring(2);
```

Comment:
Does this work for Custom Elements? iirc we're laking test cases for them. /cc @andrewiggins @developit

Reply:
It does! This is better than what I had here.

One nit:  this breaks in Workers. Maybe we should replace `window` with `self`?

```suggestion
		name = (nameLower in self ? nameLower : name).substring(2);
```

Reply:
Another option we could consider is using a sigil:

```js
<div onAnimationEnd={..} />  // gets lowercase'd
<div on-animationEnd={..} />  // addEventListener('animationEnd', ..)
```

Thoughts?

Reply:
Uh, nice suggestion! :) using `self` is a tad more _decoupled_ 😄

Not sure about the sigil though. It seems that custom elements does/should support events with dash in them, so one could actually _name_ the event with sigil as well, and this worries me that would lead to much more confusion than clarity? It's another different char than dash, but looks like a dash? 😄